app/root.tsx

```diff
@@ -133,3 +129,49 @@ export function ErrorBoundary() {
         </ContentWrapper>
     );
 }
+
+export const meta: MetaFunction = () => {
+    const title = 'ReClaim: Home Goods Store';
+    const description = 'Essential home products for sustainable living';
+    const imagePath = '/cover.jpg';
```

Comment:
How about `open-graph-image.jpg` or `social-media-image.jpg` instead of nondescript `cover.jog`?